capitalone/rubicon-ml
# add new quick look

closes: #210 

---

## What
  * uses the example from the demo notebook to replace the old quick look
    * breaks down quick look into three parts - logging, sharing, and visualizing
    * removes the old quick look notebook as well as the logging basics ones, which the new logging quick look covers

## How to Test
  * build the docs locally, proofread the new additions for typos, make sure there are no dead links to the old quick look
